docs/_data/portland-2021-schedule.yaml

```diff
@@ -0,0 +1,79 @@
+writing_day:
+  - time: '8:00'
+    title: 'Platform opens'
+  - time: '9:30'
+    title: Morning introduction - Introduction to Sprints & Project Introductions
+  - time: '9:45'
+    title: Team up and start working
+  - time: '12:30'
+    title: Lunch Break
+  - time: '5:00'
+    title: End of Writing Day
```

Comment:
We should check with @canncrochet what are the WW tour/orientation plans and put them on the schedule (Hi Christy! Any input here?)